### ISSUE TITLE
How to intercept transporter errors ? 

### ISSUE THREAD

**Issue by stanislav-web** [moleculerjs/moleculer#608]
## Prerequisites

Please answer the following questions for yourself before submitting an issue. 

- [ ] I am running the latest version (0.14-beta)
- [ ] I checked the documentation and found no answer

## Current Behavior
Anybody know how to intercept Transporter error inside source ? I am using Molecular Web [API] and no one onError interceptor doesn't catch transporter logs.  I need to report this events into Google Stackdriver
<img width="1056" alt="Снимок экрана 2019-11-04 в 0 48 20" src="https://user-images.githubusercontent.com/5216760/68093350-76534500-fe9d-11e9-934a-3da90f3b276e.png">

### Here is my transporter config (i just want to intercept any transporter events from my services)
```js
const broker = new ServiceBroker({
    transporter: {
    type: 'NATS',
    options: {
      url: process.env.TRANSPORTER_URL,
      user: process.env.TRANSPORTER_USER,
      pass: process.env.TRANSPORTER_PASS
    }
});
```

### Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Moleculer version: 0.14.0-beta5
* NodeJS version: 10.16.3
* Operating System: Mac OS

**Comment by icebob** [comment]
Currently it is not available to catch transporter errors.

**Comment by mike-aungsan** [comment]
+1

**Comment by intech** [comment]
Released in [v0.14.19](https://github.com/moleculerjs/moleculer/releases/tag/v0.14.19) by PR #1048
